website/content/api-docs/secret/ssh.mdx

```diff
@@ -885,13 +885,16 @@ $ curl \
 }
 ```
 
-## Generate Certificate and Key
+## Generate Certificate and Key (starting from Vault v1.12.0)
```

Comment:
I wouldn't add this to the title as it impacts the internal anchors and anyone who has sent a link to this will now have it not go anywhere. :/

I'd probably add another note with this info.